main/etc/systemd/system/cockpit.service

```diff
@@ -1,15 +1,34 @@
 [Unit]
-Description=quay.io/cockpit/ws:latest podman container
+Description=Cockpit Container
 After=network-online.target
 Wants=network-online.target
+SourcePath=/etc/containers/systemd/cockpit.container
+RequiresMountsFor=%t/containers
+RequiresMountsFor=/
+
+[X-Container]
+Image=quay.io/cockpit/ws:latest
+ContainerName=cockpit-ws
+Environment=NAME=cockpit-ws
+
+#flag for autoupdates
+Label=io.containers.autoupdate=registry
+
+Volume=/:/host
+PodmanArgs=--pid host --privileged
+Exec=/container/label-run
 
 [Service]
-Type=oneshot
-ExecStartPre=-/usr/bin/podman rm -f cockpit-ws
-ExecStart=/usr/bin/podman container runlabel --name cockpit-ws RUN quay.io/cockpit/ws:latest
-ExecStop=/usr/bin/podman rm -f cockpit-ws
-RemainAfterExit=yes
-KillMode=none
+Restart=always
+Environment=PODMAN_SYSTEMD_UNIT=%n
+KillMode=mixed
+ExecStopPost=-/usr/bin/podman rm -f -i --cidfile=%t/%N.cid
+ExecStopPost=-rm -f %t/%N.cid
+Delegate=yes
+Type=notify
+NotifyAccess=all
+SyslogIdentifier=%N
+ExecStart=/usr/bin/podman run --name=cockpit-ws --cidfile=%t/%N.cid --replace --rm --log-driver passthrough --cgroups=split --sdnotify=conmon -d ->
```

Comment:
@Nitrousoxide 
the `ExecStart=`  was failing for me in testing this too... I think i see what happened  `-d ->` looks like the output got cut off by `systemctl cat` not line-wrapping.

Correct line should be:
```
ExecStart=/usr/bin/podman run --name=ws --cidfile=%t/%N.cid --replace --rm --cgroups=split --sdnotify=conmon -d -v /:/host --env NAME=ws --label io.containers.autoupdate=registry --pid host --privileged quay.io/cockpit/ws:latest /container/label-run
```